src/app/views/ChannelsList.svelte

```diff
@@ -39,6 +39,8 @@
     }
   })
 
+  const markAllChannelsRead = () => setChecked($channels.map(c => c.id))
```

Comment:
Let's do something here like `channels/*`, since old events frequently come in after marking all as read, resetting the notification bell. This is a flaw of the current approach.